tools/core_ml/README.md

```diff
@@ -0,0 +1,70 @@
+# Convert MXNet models into Apple CoreML format.
+
+This tool helps convert MXNet models into [Apple CoreML](https://developer.apple.com/documentation/coreml) format which can then be run on Apple devices.
+
+## Installation
+In order to use this tool you need to have these installed:
+* mxnet 0.10.0. [Installation instructions](http://mxnet.io/get_started/install.html).
+* python 2.7
+* coremltools 0.4.0 (pip install coremltools)
+* yaml (pip install pyyaml)
+
+## How to use
+Let's say you want to use your MXNet model in an iPhone App. For the purpose of this example, let's say you want to use squeezenet-v1.1.
+
+1. Download the model into the directory where this converter resides. Squeezenet can be downloaded from [here](http://data.mxnet.io/models/imagenet/squeezenet/).
```

Comment:
Having to copy model to the convertor location is not a good idea. In some situations you may not even have permissions to do that. Can you look into the suggestion @zhreshold made above?